### PR TITLE
Marketplace shop settings

### PR DESCRIPTION
**Pull request by lcampanis**
**Please review and merge https://github.com/reactioncommerce/reaction/pull/1934 before this PR.**

This PR is mainly a tidy up of shop settings for sellers.

At the moment `Shop settings` sit in the user Profile page, this was mainly for simplicity.

In this PR we have:
1. Moved Seller Shop settings to sidebar Action View, accessible via `Shop Settings` shortcut
1. Added `Shop Address` fields
3. Added `Base Units of Measure` to shop settings
3. Added `Timezone` to shop settings
2. Added audience `seller` permissions to `Orders` shortcut

The following results are produced from this PR:
# User Dropdown shortcuts
image

# General Shop Settings
image

# Shop Address
image

### CODE REVIEW

File: imports/plugins/included/marketplace/client/templates/shops/shopSelect.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  });
+});
+
+Template.shopSelect.helpers({
+  sellerShops() {
+      const currentShopId = Reaction.Router.getParam("shopId") || 0;
+      const selector = {
+        // ignore blank site
+        _id: {
+          $ne: "ddzuN2YPvgvx7rJS5"
```
Comment (aaronjudd) [inline imports/plugins/included/marketplace/client/templates/shops/shopSelect.js]: It cannot be assumed that this is the default shop id.

File: imports/plugins/core/ui/client/components/cards/settingsCard.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Settings Card is a composite component to standardize the
+ * creation settings cards (panels) in the dashboard.
+ */
+
+import React, { Component, PropTypes } from "react";
+import Blaze from "meteor/gadicc:blaze-react-component";
+import { Card, CardHeader, CardBody } from "/imports/plugins/core/ui/client/components";
+
+class SettingsCard extends Component {
```
Comment (aaronjudd) [inline imports/plugins/core/ui/client/components/cards/settingsCard.js]: @mikemurray are we working on something similar? I think we need to do a bit of gap/review here?

File: imports/plugins/included/marketplace/client/templates/shops/shopSelect.js (hunk 1 above)
Comment (lcampanis) [inline imports/plugins/included/marketplace/client/templates/shops/shopSelect.js]: We have be updated this publication with observers now, which allow for transforms in seller data and are no longer ignoring any id. Will create a PR by tomorrow with that, and other core changes.

File: imports/plugins/core/ui/client/components/cards/settingsCard.js (hunk 2 above)
Comment (lcampanis) [inline imports/plugins/core/ui/client/components/cards/settingsCard.js]: This most probably is RC code here. We can update this PR if you think it fits RC's requirements from the description and screenshots. It's just a cleaner version of the shop > shop settings, and without the additional owner stuff.

File: imports/plugins/included/marketplace/client/templates/shops/shopSelect.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  this.autorun(() => {
+    // watch path change to reset toggle
+    Reaction.Router.watchPathChange();
+  });
+});
+
+Template.shopSelect.helpers({
+  sellerShops() {
+      const currentShopId = Reaction.Router.getParam("shopId") || 0;
+      const selector = {
```
Comment (spencern) [inline imports/plugins/included/marketplace/client/templates/shops/shopSelect.js]: Add TODO: to remove this hardcoded blank site ignorer.

**Pull request merged**